src/Api/Service/Sf.php

```diff
@@ -369,6 +372,18 @@ public function project_getJoinRequests()
         return ProjectCommands::getJoinRequests($this->projectId);
     }
 
+    public function project_getLanguage()
```

Comment:
This API method (and the one below it) is unnecessary. Use the **lex_projectDto** below and let the front end extract out what it needs itself. You'll quickly find you need lots of items from it anyway and it will only hit the server once rather than every time it needs a little bit of info.

Besides **ProjectCommands::readProject**, amongst other things, doesn't encode dates correctly. Which is why the Lexicon app uses the existing **lex_projectDto** method. Most of the time you should be able to use existing API methods rather than creating new ones.